src/frontend/scss/vendor/_icomoon.scss

```diff
@@ -0,0 +1,47 @@
+$icomoon-font-family: "icomoon" !default;
```

Comment:
I think this is some excessive parameterization ported over from Richie, which we should probably not replicate here 😇

Reply:
I cioy/paste it from the icomoon generated code :stuck_out_tongue: I should remove this variable and use direcly `icomoon` after ?

Reply:
I think so

Reply:
:ok_hand: